doc/index.rst

```diff
@@ -1,78 +1,87 @@
-PennyLane Strawberry Fields Plugin
+PennyLane-Strawberry Fields Plugin
 ##################################
 
 :Release: |release|
-:Date: |today|
 
+.. image:: _static/puzzle_sf.png
+    :align: center
+    :width: 70%
+    :target: javascript:void(0);
 
-This PennyLane plugin allows the Strawberry Fields simulators to be used as PennyLane devices.
+|
 
-`Strawberry Fields <https://strawberryfields.readthedocs.io>`_ is a full-stack Python library for designing, simulating, and optimizing continuous variable (CV) quantum optical circuits.
+.. include:: ../README.rst
+  :start-after:	header-start-inclusion-marker-do-not-remove
+  :end-before: header-end-inclusion-marker-do-not-remove
 
-`PennyLane <https://pennylane.readthedocs.io>`_ is a machine learning library for optimization and automatic differentiation of hybrid quantum-classical computations.
 
+Once the PennyLane-SF plugin is installed, the two provided Strawberry Fields devices can be accessed
+straight away in PennyLane, without the need to import any additional packages.
 
-Features
-========
-
-* Provides two devices to be used with PennyLane: ``strawberryfields.fock`` and ``strawberryfields.gaussian``. These provide access to the Strawberry Fields Fock and Gaussian backends respectively.
-
+Devices
+=======
 
-* Supports all core PennyLane operations and observables across the two devices.
+PennyLane-SF provides two Strawberry Fields devices for PennyLane:
 
+.. devicegalleryitem::
+    :name: 'strawberryfields.fock'
+    :description: Full simulator that supports all continuous-variable operations.
+    :link: devices/fock.html
 
-* Combine Strawberry Fields' advanced quantum simulator suite with PennyLane's automatic differentiation and optimization.
+.. devicegalleryitem::
+    :name: 'strawberryfields.gaussian'
+    :description: Optimized simulator that supports only Gaussian operations.
```

Comment:
```suggestion
    :description: Optimized simulator that supports only Gaussian operations and photon
        number resolving measurements.
```
(Since PNR measurements are non-gaussian)